### PR TITLE
session: upgrade mysql.db user column length to 32

### PR DESCRIPTION
**Pull request by tiancaiamao**
To keep consistent with MySQL, so we won't meet this error:

```
tidb(127.0.0.1)> grant all privileges on test.* to 'abcddfjakldfjaldddds'@'%' identified by '';
ERROR 1406 (22001): Data too long for column 'User' at row 1
```

@XuHuaiyu @coocood

### CODE REVIEW

**Comment by coocood** [comment]
LGTM

File: session/session_test.go [hunk 1]
```diff
@@ -1975,3 +1975,11 @@ func (s *testSessionSuite) TestRollbackOnCompileError(c *C) {
 	}
 	c.Assert(recoverErr, IsTrue)
 }
+
+func (s *testSessionSuite) TestDBUserNameLength(c *C) {
+	tk := testkit.NewTestKitWithInit(c, s.store)
+	tk.MustExec("create table if not exists t (a int)")
+	// Test user name lengh can be longer than 16.
```
Comment (zimulala) [inline session/session_test.go]: s/lengh/length

**Pull request merged**